pkgs/by-name/ls/lsd/package.nix

```diff
@@ -1,37 +1,32 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-, fetchpatch
-, rustPlatform
-, installShellFiles
-, darwin
-, pandoc
-, testers
-, lsd
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  rustPlatform,
+  installShellFiles,
+  darwin,
+  pandoc,
+  testers,
+  lsd,
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "lsd";
-  version = "1.1.2";
+  version = "1.1.3";
 
   src = fetchFromGitHub {
     owner = "lsd-rs";
     repo = "lsd";
     rev = "v${version}";
-    hash = "sha256-ZMaI0Q/xmYJHWvU4Tha+XVV55zKLukrqkROfBzu/JsQ=";
+    hash = "sha256-olOoGBgNGP5NjKARU1TN0YW2cCsjkon8gKmBtsOMVS4=";
```

Comment:
Somehow the hash changed again

From ofborg sha256-4vf68Rga2Z+PYIdfzzmDtfjCQVbr8RWiVG29c/UDVsQ=